app/schemas/USER.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+      "userId": {
+        "type": "string"
+      },
+      "f_name": {
+        "type": "string"
+      },
+      "l_name": {
+        "type": "string"
+      },
+      "email": {
+        "type": "string"
+      },
+      "phone_number": {
+        "type": "string"
+      },
+      "location": {
+        "type": "object",
+        "properties": {
+          "latitude": {
+            "type": "number"
+          },
+          "longitude": {
+            "type": "number"
+          }
+        },
+        "required": [
+          "latitude",
+          "longitude"
+        ]
+      },
+      "address": {
```

Comment:
Do we need an address if we're saving the lat and long? Especially since we're not going to be placing the actual food order, we're going to be redirecting to the location the user can buy the food